deploy/Dockerfile_perf

```diff
@@ -0,0 +1,27 @@
+FROM alpine:3.10
+
+RUN apk --no-cache add libc6-compat=1.1.22-r3 device-mapper=2.02.184-r0 findutils=4.6.0-r1 zfs=0.8.2-r0 build-base=0.5-r1 linux-headers=4.19.36-r0 && \
+    apk --no-cache add thin-provisioning-tools=0.7.1-r3 --repository http://dl-3.alpinelinux.org/alpine/edge/main/ && \ 
+    echo 'hosts: files mdns4_minimal [NOTFOUND=return] dns mdns4' >> /etc/nsswitch.conf && \
+    rm -rf /var/cache/apk/*
+
+
+RUN wget https://sourceforge.net/projects/perfmon2/files/libpfm4/libpfm-4.10.1.tar.gz && \
```

Comment:
Is there a reason why this shouldn't just be the "normal" build of cAdvisor?  Would some users not want this installed, or does it come with any downsides?

Reply:
In case of non-containerized cAdvisor perf requires users to install one package (available on Debian and Centos). It would be great to have it enabled by default and it could be really beneficial for users.
I'm not sure how keen the users would be (especially taking that cAdvisor is a Kubernetes dependency...)

Reply:
But the Dockerfile is only used for containerized users

Reply:
Then it sound like a very good idea to me.

Reply:
> Is there a reason why this shouldn't just be the "normal" build of cAdvisor? Would some users not want this installed, or does it come with any downsides?

I don't see downsides of having libpfm in normal build of cAdvisor. If cAdvisor binary is built without libpfm then introduced changes don't have any impact. 

If cAdvisor binary is built with  libpfm according to [this instruction](https://github.com/google/cadvisor/blob/master/docs/development/build.md#perf-support) then introduced changes enable of having perf metric exposed on API and on metric endpoint.

Changes to normal Dockerfile are visible in [this commit](https://github.com/google/cadvisor/pull/2555/commits/cd46e074899a6e131b35f52b1983c6d01b878e27).